code/modules/hydroponics/grown/nymph.dm

```diff
@@ -12,6 +12,37 @@
 	yield = 1
 	reagents_add = list("plantmatter" = 0.1)
 
+// If the nymph pods are fully grown, allow ghosts that click on them to spawn as a nymph (decreases yield by 1 and kills the plant when yield reaches 0)
+/obj/item/seeds/nymph/GhostAttackWhenPlanted(mob/dead/observer/O, obj/machinery/hydroponics/T)
```

Comment:
why make an entirely new proc when this can literally be `attack_ghost`

Reply:
My reasoning at the time was that I wanted ghost_attack on the seed packet and on the hydroponics tray itself to be different procs, the ultimate idea being to make a framing for other types of plant to potentially be ghost-interactable. In retrospect though I could probably do better by getting the tray and plant status from the seed packet itself and checking based on that. I'll make some adjustments.